Source/Scene/Scene.js

```diff
@@ -3334,6 +3354,12 @@ function updateAndClearFramebuffers(scene, passState, clearColor) {
   const picking = passes.pick;
   const useWebVR = environmentState.useWebVR;
 
+  const originalSamples = scene.numberSamples;
+  if (picking) {
+    scene.numberSamples = 1;
+    scene.samplesDirty = view.oit.numSamples !== scene.numberSamples;
```

Comment:
I'm not a fan of this change, but I added it because disabling multisampling during pick frames broke OIT. The idea is: 

1. Frames get rendered with multisampling
2. `Picking` requests a frame, during which multisampling is disabled during updates and then reenabled before the next frame. The non-MSAA frame skips OIT's update.
3. On the next non-pick frame, OIT thinks that nothing changed because it has been seeing the same number of samples all along. But, GlobeDepth has actually made new attachments because it knows that it just switched from 1 to >1 samples, and OIT's depth stencil texture is outdated.

Though the `samplesDirty` check fixes invert classification and other OIT Sandcastles I tested, it breaks [Globe Translucency](http://localhost:8080/Apps/Sandcastle/index.html?src=Globe%20Translucency.html) - not sure why yet. @lilleyse I'm still unsure if disabling reenabling multisampling in this function is the right way to go about it - any thoughts on this approach?

Reply:
Another downside here is that this will destroy and recreate framebuffers and their attachments every time Scene switches from a normal frame to a pick frame when MSAA is on. Maybe with MSAA on it makes sense for GlobeDepth to always have a single-sample `_colorFramebuffer` that's only used during pick frames to avoid this.